### DISCUSSION TITLE
Help to understand script for a Shelly Dimmer 2

### DISCUSSION THREAD

**Discussion by just-doit** [RavenSystem/esp-homekit-devices#2263] [MEPLHAA Scripts]
I have recently flashed a Shelly Dimmer 2 and am now trying to configure a suitable script.
I came across the script in the wiki database (https://github.com/RavenSystem/esp-homekit-devices/wiki/devices-database).
On the other hand, you can also find a script here (https://github.com/RavenSystem/esp-homekit-devices/discussions/1551).

What I'm wondering in this context is why the UART commands are different.
For example, the script in the wiki database says: (Boot action, if I understand correctly)
`“0”:{“u”:[{“v”: “010202060000f401000000ff04”}]},`  
while here in the discussions it is  
`“0”:{“u”:[{“v”: “01cc0202000000d004”}]},`

Is there any documentation on what the UART data means?

**Comment by RavenSystem** [comment]
Better, ask @Oo80 at https://github.com/RavenSystem/esp-homekit-devices/discussions/1551 because its script seems to be better.
